### ISSUE TITLE
Movie watched once separated into pieces

### ISSUE THREAD

**Issue by bttd** [CyferShepard/Jellystat#56]
Hi,

If I watch a movie or series, sometimes when the device has poor network connectivity, the system breaks it down into segments lasting a few minutes. This makes it seem as if I have watched the particular movie or episode of a series 25 times.

![jellystat](https://github.com/CyferShepard/Jellystat/assets/45686509/943066d1-6759-4c46-8d89-9a5096f6ecfb)

**Comment by CyferShepard** [comment]
Hi @bttd , thanks for bringing this to my attention, i never had this issue, but il look into a way to add a buffer to prevent this from happening

**Comment by bttd** [comment]
I can see the same amount of playback under the activity tab in Jellyfin too. I'm going to report this to Jellyfin devs too.

Regards

**Comment by CyferShepard** [comment]
Hmm, if its also happening in Jellyfin then it may be an issue on their side, if it drops the session due to network issues the api reports it as a stop start action. Il still look for a way to accommodate it but lets see what the Jellyfin devs say aswell

**Comment by ados8** [comment]
I would definitely like to bump this.
When users start and stop a movie in short periods it shows as "watched".
Tautulli worked used percentage which could be adjusted.
This for example allowed for say 10% played for it to be considered watched.
I guess another self check could be if you start and stop the same content in sequence it considers it a single play as long as the percentage level has been satisfied.

**Comment by CyferShepard** [comment]
![](https://github.trello.services/images/mini-trello-icon.png) [Progressively Track Watch instance](https://trello.com/c/z0of9Nrf/9-progressively-track-watch-instance)

**Comment by geebru** [comment]
> I can see the same amount of playback under the activity tab in Jellyfin too. I'm going to report this to Jellyfin devs too.
> 
> Regards

Did you submit this bug and hear back? I saw similar activity on Jellyfin - one user had 28 plays on a show with 11 episodes as she had some issues getting her AirPlay to work and it counted each "try" as a unique play. I watched a movie last night and did 2 restarts for subtitle reasons, and once it went to screensaver, and it counted as 4 plays.

I think Jellystat could get around this with an option on how long to watch consecutively to count a 'play' but I agree it's aggressive from Jellyfins side of things.
